modules/ting_smart_search/ting_smart_search.module

```diff
@@ -550,3 +557,52 @@ function ting_smart_search_search_stats_importer($search_data) {
     watchdog_exception('ting_smart_search', $e, 'Importing search feed data to database failed');
   }
 }
+
+/**
+ * Exposes smart search records as a json array.
+ */
+function ting_smart_search_records() {
+  $data = ting_smart_search_get_records();
+  if (isset($data)) {
+    $json = json_encode($data, JSON_PRETTY_PRINT);
+    printf("<pre>%s</pre>", $json);
```

Comment:
Wrapping JSON in HTML tags should not work. Please use [`drupal_json_output()`](https://api.drupal.org/api/drupal/includes%21common.inc/function/drupal_json_output/7.x).